backend/controller/cronjobs/cronjobs.go

```diff
@@ -307,7 +307,7 @@ func (s *Service) watchForUpdates(ctx context.Context) {
 		now := s.clock.Now()
 		next := now.Add(time.Hour) // should never be reached, expect a different signal long beforehand
 		for _, j := range state.jobs {
-			if possibleNext, err := s.nextAttemptForJob(j, state, false); err == nil && possibleNext.Before(next) {
+			if possibleNext, err := s.nextAttemptForJob(j, state, now,false); err == nil && possibleNext.Before(next) {
```

Comment:
You might need to setup your editor/formatter it looks like. This type of stuff `state, now,false` should autofix when you save